.gitignore

```diff
@@ -20,6 +20,9 @@ none
 
 # Compiled CSS
 /assets/css/*.css
+/assets/css/photoswipe/*.min.css
```

Comment:
Photoswipe can stay since they do not include minified files but sass files which we compile.